.prettierrc

```diff
@@ -4,5 +4,5 @@
   "trailingComma": "all",
   "singleQuote": true,
   "semi": false,
-  "endOfLine": "auto"
+  "endOfLine": "lf"
```

Comment:
I think it could also work if we remove this completely and the default takes effect.